src/lit-html.ts

```diff
@@ -159,7 +163,7 @@ export function render(
     container: Element|DocumentFragment,
```

Comment:
Why not make TemplateContainer the type of `container` here? It'll still be legal to pass in any Element or DocumentFragment

Reply:
Ah, that's true because the property is optional. Still, this is an internal detail of the render function. I don't want the type to change if we used a WeakMap instead of an expando property.